src/Uno.Extensions.Configuration/ConfigBuilderExtensions.cs

```diff
@@ -146,7 +146,10 @@ public static IConfigBuilder EmbeddedSource<TApplicationRoot>(this IConfigBuilde
 			.ConfigureAppConfiguration((ctx, b) =>
 			{
 				var path = FilePath(ctx);
-				b.AddJsonFile(path, optional: true, reloadOnChange: false); // In .NET6 we can enable this again because we can use polling
+				if (path is { Length: > 0 })
+				{
+					b.AddJsonFile(path, optional: true, reloadOnChange: false); // In .NET6 we can enable this again because we can use polling
```

Comment:
Do we need this comment?

Reply:
I left it because at some point need to revisit the way we're refreshing on changes and potentially where applicable allow reloadOnChange to be true